client/src/components/messenger/Messenger.js

```diff
@@ -16,36 +17,57 @@ const Messenger = () => {
     const dispatch = useDispatch();
 
     const [currentConvo, setCurrentConvo] = useState({});
-    const changeConvo = (id, title) => {
-        setCurrentConvo({ id, title });
+    const changeConvo = (id, title, participants) => {
+        setCurrentConvo({ id, title, participants });
     };
 
     const userInfo = useSelector((state) => state.user.userInfo);
+    const token = useSelector((state) => state.user.token);
 
     const updateUser = (userInfo) => dispatch(setUserInfo(userInfo));
 
+    const [socket] = useState(() => {
+        let soc = io("http://localhost:5000", {
```

Comment:
Why do we need the full address here? This should be relative no?

Reply:
@D-Garnes

Reply:
The current react proxy is not working with the socket.io that is why the full address was used. I'll sort it out.